### PR TITLE
Dark Mode Color Tweaks

### PR DESCRIPTION
**Pull request by russetdapone**
## What Does This PR Do
Changes the color scheme for the GoonChat Dark-mode to a a hybridization of TG-Station and Paradise Station, making Dark Mode easier to read without touching Light mode. All Radios are changed for either more saturation or better contrast, and the default color is changed to an off-white color.

## Why It's Good For The Game
It both looks better, and feels better to use. It helps users who have Dyslexia with reading the colors such as Tips, otherwise known as GDaws696#2388. 
image
image

## Images of changes
image

## Changelog
:cl: Alexios
Tweak: Changed Dark Mode Colors for readability.
/:cl:

### CODE REVIEW

**Review by AffectedArc07 — COMMENTED** [review]
You uh, uploaded this file to the repo root instead of editing the correct file

**Comment by russetdapone** [comment]
> 

_Yikes how the fuck did I manage that_

File: goon/browserassets/css/browserOutput-dark.css [hunk 1]
```diff
@@ -242,162 +242,161 @@ a.popt {text-decoration: none;}
 }
 
 /* OUTPUT COLORS */
-.highlight {background: yellow;}
-h1, h2, h3, h4, h5, h6	{color: #06f;font-family: Georgia, Verdana, sans-serif;}
-em						{font-style: normal;	font-weight: bold;}
-.motd					{color: #638500;	font-family: Verdana, sans-serif;}
+.highlight				 {background-color: #ffff0026;}
```
Comment (AffectedArc07) [inline goon/browserassets/css/browserOutput-dark.css:245]: I assume the 2 extra hex-digits are for transparency?

Comment (russetdapone) [reply goon/browserassets/css/browserOutput-dark.css:245]: They don't seem to affect anything, nor harm anything, It seems like its RGB only, no RGBA, which is frustrating.

**Comment by S34NW** [comment]
Our eyes no longer bleed, thank you

**Comment by S34NW** [comment]
How does AI Private stack up against procedure? And do the various others not in the screenshot, such as Syndie and CC comms?

File: goon/browserassets/css/browserOutput-dark.css (hunk 1 above)
Comment (Sirryan2002) [reply goon/browserassets/css/browserOutput-dark.css:245]: can we get a confirmation on this not affecting things in-game? If I remember correctly, the alpha channel for hex colors aren't supported by IE11 for background-color.

File: goon/browserassets/css/browserOutput-dark.css [hunk 2]
```diff
@@ -242,162 +242,161 @@ a.popt {text-decoration: none;}
...
 
-.revennotice				{color: #6685F5;}
+.revennotice			{color: #6685F5;}
 .revenboldnotice			{color: #6685F5;	font-weight: bold;}
-.revenbignotice				{color: #6685F5;	font-weight: bold; font-size: 120%;}
-.revenminor				{color: #823abb}
-.revenwarning				{color: #760fbb;	font-style: italic;}
-.revendanger				{color: #760fbb;	font-weight: bold; font-size: 120%;}
+.revenbignotice			{color: #6685F5;	font-weight: bold; font-size: 120%;}
+.revenminor			{color: #823abb}
```
Comment (Sirryan2002) [inline goon/browserassets/css/browserOutput-dark.css:382]:
```suggestion
.revenminor			{color: #823abb;}
```
inconsistent attribute ending

File: goon/browserassets/css/browserOutput-dark.css (hunk 1 above)
Comment (russetdapone) [reply goon/browserassets/css/browserOutput-dark.css:245]: Highlight affects CSS, all it will do is not use the values.

**Comment by warriorstar-orion** [comment]
Service and Common still feel pretty similar, maybe Service could be desaturated a bit more?

**Comment by russetdapone** [comment]
More updates! 
image

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**